src/components/BallotProxyMetadata.jsx

```diff
@@ -35,6 +35,8 @@ export class BallotProxyMetadata extends React.Component {
                   { value: '3', label: ballotStore.ProxyBallotType[3] },
                   { value: '4', label: ballotStore.ProxyBallotType[4] },
                   { value: '5', label: ballotStore.ProxyBallotType[5] },
+                  { value: '7', label: ballotStore.ProxyBallotType[7] },
```

Comment:
Why do we skip 6th index here?

Reply:
I took indexes from ProxyStorage.sol:

```
enum ContractTypes {
    Invalid,
    KeysManager,
    VotingToChangeKeys,
    VotingToChangeMinThreshold,
    VotingToChangeProxy,
    BallotsStorage, 
    PoaConsensus,
    ValidatorMetadata,
    ValidatorMetadataEternalStorage
}
```

i.e. 6th index is for `PoaConsensus`.